Build issue

c:\Dominic\programming\repos\git\xi-win\xi-win-shell>cargo build
   Compiling winapi-x86_64-pc-windows-gnu v0.4.0
   Compiling winapi v0.3.6
   Compiling num-traits v0.2.6
   Compiling num-integer v0.1.39
   Compiling rustc-serialize v0.3.24
   Compiling num-iter v0.1.37
   Compiling version_check v0.1.5
   Compiling boolinator v2.4.0
   Compiling either v1.5.0
   Compiling libc v0.2.43
   Compiling lazy_static v1.1.0
   Compiling num-complex v0.1.43
   Compiling rand v0.4.3
   Compiling wio v0.2.0
   Compiling time v0.1.40
   Compiling directwrite v0.1.4
   Compiling num-bigint v0.1.44
   Compiling num-rational v0.1.42
   Compiling num v0.1.42
   Compiling dxgi v0.1.7
   Compiling direct2d v0.2.0
   Compiling xi-win-shell v0.1.0 (C:\Dominic\programming\repos\git\xi-win\xi-win-shell)
error[E0432]: unresolved import `winapi::um::handleapi`
  --> src\util.rs:33:17
   |
33 | use winapi::um::handleapi::*;
   |                 ^^^^^^^^^ Could not find `handleapi` in `um`

error[E0425]: cannot find value `INVALID_HANDLE_VALUE` in this scope
   --> src\util.rs:275:22
    |
275 |         if stdout != INVALID_HANDLE_VALUE && GetFileType(stdout) != FILE_TYPE_UNKNOWN {
    |                      ^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `INVALID_HANDLE_VALUE` in this scope
   --> src\util.rs:291:24
    |
291 |             if chnd == INVALID_HANDLE_VALUE {
    |                        ^^^^^^^^^^^^^^^^^^^^ not found in this scope

warning: unused import: `winapi::um::handleapi::*`
  --> src\util.rs:33:5
   |
33 | use winapi::um::handleapi::*;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: #[warn(unused_imports)] on by default

error: aborting due to 3 previous errors

Some errors occurred: E0425, E0432.
For more information about an error, try `rustc --explain E0425`.
error: Could not compile `xi-win-shell`.

To learn more, run the command again with --verbose.

c:\Dominic\programming\repos\git\xi-win\xi-win-shell>

--- a/xi-win-shell/Cargo.toml
+++ b/xi-win-shell/Cargo.toml
@@ -15,4 +15,4 @@ time = "0.1.39"
 
 [dependencies.winapi]
 version = "0.3.6"
-features = ["d2d1_1", "dwrite", "winbase", "libloaderapi", "errhandlingapi", "winuser", "shellscalingapi", "shobjidl", "combaseapi", "synchapi", "dxgi1_3", "dcomp", "d3d11", "dwmapi", "wincon", "fileapi", "processenv", "winbase"]
+features = ["d2d1_1", "dwrite", "winbase", "libloaderapi", "errhandlingapi", "winuser", "shellscalingapi", "shobjidl", "combaseapi", "synchapi", "dxgi1_3", "dcomp", "d3d11", "dwmapi", "wincon", "fileapi", "processenv", "winbase", "handleapi"]

